### ISSUE TITLE
Uninstall says it cannot find manifest file

### ISSUE THREAD

**Issue by uglyeoin** [techjoomla/com_tjucm#198]
Don't seem to be able to uninstall this for some reason.

**Comment by ankush-maherwal** [comment]
@uglyeoin Yes, it seems that there is some issue in uninstalling the package.

You can uninstall the component as of now.

We will check this issue and will fix in the upcoming releases
